### PR TITLE
Bug with sshd

### LINKED ISSUE #1294: [Bug]: Docker container does not start
### Is there an existing issue for the same bug?

- [X] I have checked the troubleshooting document at https://github.com/OpenDevin/OpenDevin/blob/main/docs/guides/Troubleshooting.md
- [X] I have checked the existing issues.

### Describe the bug

Backend can't start the container because 
`ssh_box.py:325 - b'Missing privilege separation directory: /run/sshd\r\n'`

### Current Version

```bash
ghcr.io/opendevin/sandbox:latest
```

### Installation and Configuration

```bash
make build
make start-backend
make start-frontend
```

### Model and Agent

Does not seem to make a difference.

### Reproduction Steps

_No response_

### Logs, Errors, Screenshots, and Additional Context

2024-04-22 at 15 04 09

### PR DESCRIPTION
**Pull request by enyst**
Got an error when trying to run the sandbox today:
ssh_box.py:325 - b'Missing privilege separation directory: /run/sshd\r\n'

There is a bug with the sandbox, and rebuilding it with the ssh service start line, for no apparent reason, worked. (forcing sshd to make that directory, presumably)

Needs confirmation, it's a weird one, and I also did other things when my system didn't want to start opendevin. 

https://askubuntu.com/questions/1110828/ssh-failed-to-start-missing-privilege-separation-directory-var-run-sshd

https://askubuntu.com/questions/1109934/ssh-server-stops-working-after-reboot-caused-by-missing-var-run-sshd/1110843#1110843

May fix #1294

### CODE REVIEW

File: containers/sandbox/Dockerfile [hunk 1]
```diff
@@ -18,5 +18,7 @@ RUN apt-get update && apt-get install -y \
     sudo \
     && rm -rf /var/lib/apt/lists/*
 
+RUN service ssh start
```
Comment (enyst) [inline containers/sandbox/Dockerfile:21]:
```suggestion
RUN mkdir -p -m0755 /var/run/sshd
```

**Comment by xingyaoww** [comment]
This is super weird... Let me try to reproduce and test the updated Dockerfile, with try to get this fixed asap

**Comment by enyst** [comment]
Admittedly, "this line is useless" has a good record in famous last words! 😅

**Comment by xingyaoww** [comment]
LOL! Just ran lolcally and confirm that replacing `RUN service ssh start` with `RUN mkdir -p -m0755 /var/run/sshd` works! At least we are making some progresses after we say that last word ;)

**Review by xingyaoww — APPROVED** [review]
LGTM

**Pull request merged**